wagtail/search/management/commands/update_index.py

```diff
@@ -135,6 +140,8 @@ def add_arguments(self, parser):
         )
 
     def handle(self, **options):
+        self.verbosity = options["verbosity"]
```

Comment:
We should preserve the current behaviour where this command does output OR we should ensure the docs indicate the change. 

Not sure on Django's conventions here but we should probably respect that. From a quick search it looks like they always default it to `1`. 

https://github.com/django/django/blob/93d4c9ea1de24eb391cb2b3561b6703fd46374df/django/db/backends/base/creation.py#L33

Reply:
I could be wrong but i believe this is dealt with here:
https://github.com/django/django/blob/main/django/core/management/base.py#L304-L315
So defaults to 1 (as with all commands) - i dont _think_ we need anything else here (unless we want to be explicit in the docs that we're respecting the arguements.)

Reply:
good find - thank you.